### ISSUE TITLE
Proper use of --force argument

### ISSUE THREAD

**Issue by czalidis** [pandora-auth-ros-pkg/pandoradep#2]
When trying to run `pandoradep` on [pandora_visualization][1] metapackage (branch: geotiff), I get an error:

```
Package conflict in pandora_geotiff_plugins
Info: 
{'repo': 'pandora_data_fusion', 'version': None, 'name': 'pandora_data_fusion_msgs'}
{'repo': 'pandora_data_fusion', 'version': 'geotiff_interfaces', 'name': 'pandora_data_fusion_msgs'}
```

That error was initially caught on an Jenkins [job][2] and then reproduced locally.

IMO a version different that `None` should always override a `None` version without the need of `--force` argument. Otherwise, we are going to use the `--force` argument constantly, which has no point.

[1]: https://github.com/pandora-auth-ros-pkg/pandora_visualization/tree/geotiff
[2]: http://jenkins.pandora.ee.auth.gr/job/test-pandora_visualization/branch/geotiff/1/console

**Comment by mujx** [comment]
The point of not using ``--force`` by default is to catch all the conflicts and then let the user decide, in case there was a mistake. Btw a ``None`` version is the  ``hydro-devel``.

**Comment by czalidis** [comment]
Ok, let me rephrase my point. I am suggesting that when a version is trying to override the `None` version, that should not considered as a conflict.

If we consider that as a conflict, we will end up using the `--force` option at all times, which is contrary to the existence of this option.

I agree that when conflicts between versions occur (both of them different than `None`, which in our case is `hydro-devel`), an error should raised and the need of the `--force` should be mandatory in order to continue.
